cmd/cli/handler_jwk_test.go

```diff
@@ -60,4 +68,35 @@ func Test_toSDKFriendlyJSONWebKey(t *testing.T) {
 			}
 		})
 	}
+
+	conf := internal.NewConfigurationWithDefaults()
+	reg := internal.NewRegistryMemory(t, conf)
+	router := x.NewRouterPublic()
+	var testGenerator = &jwk.RS256Generator{}
+	IDKS, _ := testGenerator.Generate("test-id", "sig")
+
+	h := reg.KeyHandler()
+	require.NoError(t, reg.KeyManager().AddKeySet(context.TODO(), x.OpenIDConnectKeyName, IDKS))
+
+	h.SetRoutes(router.RouterAdmin(), router, func(h http.Handler) http.Handler {
+		return h
+	})
+	testServer := httptest.NewServer(router)
+
+	cmd := cobra.Command{
+		Use: "key",
+	}
+	cmd.Flags().String("use", "sig", "Sets the \"use\" value of the JSON Web Key if not \"use\" value was defined by the key itself")
+	cmd.Flags().Bool("fake-tls-termination", false, "Sets the \"use\" value of the JSON Web Key if not \"use\" value was defined by the key itself")
+	cmd.Flags().String("access-token", "", "Set an access token to be used in the Authorization header, defaults to environment variable OAUTH2_ACCESS_TOKEN")
+	cmd.Flags().String("endpoint", "", "Set the URL where ORY Hydra is hosted, defaults to environment variable HYDRA_ADMIN_URL. A unix socket can be set in the form unix:///path/to/socket")
+	cmd.Flags().Bool("skip-tls-verify", true, "Foolishly accept TLS certificates signed by unknown certificate authorities")
+	os.Setenv("HYDRA_URL", testServer.URL)
+	t.Run("tt.name", func(t *testing.T) {
+		NewHandler().Keys.ImportKeys(&cmd, []string{"setName", "../test/private_key.json", "../test/public_key.json"})
+		//running again to make sure the row in storage is not deleted
+		NewHandler().Keys.ImportKeys(&cmd, []string{"setName", "../test/private_key.json", "../test/public_key.json"})
+
```

Comment:
We still need to test if the keys are in the store as expected right? :)

Reply:
@aeneasr : unit test is updated, now we are checking the keys in db after key import